3rdWeekAssignment/week2/src/main/java/com/pironeer/week2/service/CommentService.java

```diff
@@ -0,0 +1,65 @@
+package com.pironeer.week2.service;
+
+import com.pironeer.week2.dto.request.CommentCreateRequest;
+import com.pironeer.week2.dto.request.CommentUpdateRequest;
+import com.pironeer.week2.dto.request.TopicCreateRequest;
+import com.pironeer.week2.dto.response.ComCommentResponse;
+import com.pironeer.week2.dto.response.CommentResponse;
+import com.pironeer.week2.dto.response.TopicResponse;
+import com.pironeer.week2.repository.ComCommentRepository;
+import com.pironeer.week2.repository.CommentRepository;
+import com.pironeer.week2.repository.TopicRepository;
+import com.pironeer.week2.repository.domain.Comment;
+import com.pironeer.week2.repository.domain.Topic;
+import lombok.RequiredArgsConstructor;
+import org.springframework.cglib.core.Local;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service
+public class CommentService {
+    private final CommentRepository commentRepository;
+    private final TopicRepository topicRepository;
+    private final ComCommentRepository comCommentRepository;
+    public void save(CommentCreateRequest request) {
+        if (topicRepository.findById(request.topicId()).isEmpty()) {
+            throw new RuntimeException("TOPIC NOT FOUND");
+        }
```

Comment:
```suggestion
        Topic topic = topicRepository.findById(id)
                .orElseThrow(() -> new RuntimeException("TOPIC NOT FOUND"));
```

이렇게 orElseThrow를 활용하는 게 더 좋아요.